### DISCUSSION TITLE
How to change tippy appearence (color etc..) or, where to find docs about its complete markup?

### DISCUSSION THREAD

**Discussion by stratboy** [atomiks/tippyjs#1066] [General]
Hi, I think there are no docs about this, so I'm asking here:

How to change tippy appearence (color etc..) or, where to find details about its complete markup?

Thank you

**Comment by destructo570** [comment]
You can checkout the docs here. It contains all the information related creating your own theme.

https://atomiks.github.io/tippyjs/v6/themes/

If you're creating your own wrapper around it (headless) then I guess you can just use classnames to change the styling however you like.
